packages/react-server/src/ReactServerStreamConfigBrowser.js

```diff
@@ -13,6 +13,19 @@ export type PrecomputedChunk = Uint8Array;
 export opaque type Chunk = Uint8Array;
 export type BinaryChunk = Uint8Array;
 
+function handleErrorInNextTick(error: any) {
+  setTimeout(() => {
+    throw error;
+  });
+}
+
+export const scheduleMicrotask: (callback: () => void) => void =
+  typeof queueMicrotask === 'function'
+    ? queueMicrotask
+    : callback => {
+        Promise.resolve(null).then(callback).catch(handleErrorInNextTick);
+      };
```

Comment:
unlike in Fiber where there is a feature test for Promise I assume here that you must be using an environment with Promise globals. At this point I think we can safely just say React requires promises and since 19 is about to ship maybe we make this breaking change in Fiber too